imageio_ffmpeg/_definitions.py

```diff
@@ -43,12 +46,14 @@ def get_platform():
 }
 
 osxplats = "macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64"
+osxarmplats = "macosx_11_0_arm64.macosx_12_0_arm64.macosx_13_0_arm64.macosx_14_0_arm64"
```

Comment:
Can someone double check the values for both `osxplats` and `osxarmplats`, please? They are the lines I'm least confident about.

Reply:
I looked at the output of `import packaging; packaging.tags.sys_tags()` on an M1 mac, which output: tags for 11, 12, 13, and 14.

Reply:
I suspect that these can be simplified, as I think that these tags represents minimal versions. I tried to find a definite aswer [here](https://packaging.python.org/en/latest/specifications/platform-compatibility-tags/) but was not much wiser. That said the tags we use for wgpu (which also contains a precompiled binary) are simpler, and these wheels work fine:

<img width="511" alt="image" src="https://github.com/imageio/imageio-ffmpeg/assets/3015475/6a9a1308-1b4f-41fa-98f4-790e6144b0d8">

So let's change these to this:
```py
osxplats = "macosx_10_9_intel.macosx_10_9_x86_64"
osxarmplats = "macosx_11_0_arm64"
```

Reply:
> So let's change these to this:
>
> ```
> osxplats = "macosx_10_9_intel.macosx_10_9_x86_64"
> osxarmplats = "macosx_11_0_arm64"
> ```

Ok, will do